### ISSUE TITLE
A bash environment variable, HOME, is set unusally in exec.sh

### ISSUE THREAD

**Issue by byoo** [broadinstitute/cromwell#3421]
I noticed that cromwell generates exec.sh setting HOME presumably incorrectly.
This is from GATK4's somatic CNV workflow executed by cromwell 31 with JES backend. Thanks!

```
#!/bin/bash
tmpDir=$(
  set -e
  cd /cromwell_root
  tmpDir="$(mkdir -p "/cromwell_root/infant-all-cnv-out/cromwell-execution/CNVSomaticPairWorkflow/aa10c071-5c42-4f6a-a4e0-3ba96cc54283/call-ModelSegmentsTumor/tmp.3efcbfd1" && echo "/cromwell_root/infant-all-cnv-out/cromwell-execution/CNVSomaticPairWorkflow/aa10c071-5c42-4f6a-a4e0-3ba96cc54283/call-ModelSegmentsTumor/tmp.3efcbfd1")"
  echo "$tmpDir"
)
chmod 777 "$tmpDir"
export _JAVA_OPTIONS=-Djava.io.tmpdir="$tmpDir"
export TMPDIR="$tmpDir"
export HOME="$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME$HOME"
(
cd /cromwell_root

)
...
```

**Comment by mcovarr** [comment]
What version of Cromwell are you using?

**Comment by byoo** [comment]
It's 31.

**Comment by mcovarr** [comment]
@danbills This looks both bad and related to the changes I made [here](https://github.com/broadinstitute/cromwell/pull/3345) 🙁

**Comment by danbills** [comment]
@byoo fixed, will be in soon-to-be released 31.1 and next major version 32

**Comment by byoo** [comment]
Thank you for the prompt help!!
